core/modules/entity/entity.controller.inc

```diff
@@ -437,7 +442,7 @@ class DefaultEntityController implements EntityControllerInterface {
    * @param $queried_entities
    *   Associative array of query results, keyed on the entity ID.
    * @param $revision_id
-   *   ID of the revision that was loaded, or FALSE if the most current revision
+   *   ID of the revision that was loaded, or FALSE if the most default revision
```

Comment:
Let's use "active" in the code comments too.

Reply:
I saw that but didnt think it makes sense to say the "most active". That comment means the most current or most recent I think, not the most active/default/approved revision.

Reply:
Ah I see, so maybe just leave it "most current revision" as it was before. Or "most recent" if that's actually more accurate.

Reply:
I'm reading this more deeply now and I think it needs to be updated still. I think the correct phrasing is:

"ID of the revision that was loaded, or FALSE if no revision was specified."

We could add further that "If no revision is specified, the active revision is being loaded." But it's probably unnecessary for this particular method to get that in-depth on this behavior.